### PR TITLE
Bump version to v0.3.0

### PR DESCRIPTION
**Pull request by HAOCHENYE**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Review by C1rN09 — APPROVED** [review]

File: docs/en/notes/changelog.md [hunk 1]
```diff
@@ -1,5 +1,63 @@
...
+### Docs
+
+- Fix typo in `docs/zh_cn/tutorials/model.md` by @C1rN09 in https://github.com/open-mmlab/mmengine/pull/598
+- Fix typo in `docs/zh_cn/tutorials/config.md` by @Xiangxu-0103 in https://github.com/open-mmlab/mmengine/pull/596
+- Fix cannot show `changelog.md` in chinese documents. by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/606
+- Fix typo in `docs/zh_cn/advanced_tutorials/visualization.md` by @MambaWong in https://github.com/open-mmlab/mmengine/pull/616
+- Fix Chinese docs whitespaces by @C1rN09 in https://github.com/open-mmlab/mmengine/pull/521
+- Translate installation and 15_min by @xin-li-67 in https://github.com/open-mmlab/mmengine/pull/629
+- Refine chinese doc by @Tau-J in https://github.com/open-mmlab/mmengine/pull/516
+- Add mmyolo link in README by @Xiangxu-0103 in https://github.com/open-mmlab/mmengine/pull/634
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:28]: mmyolo -> MMYOLO

File: docs/en/notes/changelog.md [hunk 2]
```diff
@@ -1,5 +1,63 @@
...
+- Add a function to mark the deprecated function. by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/609
+- Call `register_all_modules`  in `Registry.get()` by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/541
+- Deprecate `_save_to_state_dict` implemented in mmengine by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/610
+- Add `ignore_keys` in ConcatDataset by @BIGWangYuDong in https://github.com/open-mmlab/mmengine/pull/556
+- Add issue or pr assignees by @zhouzaida in https://github.com/open-mmlab/mmengine/pull/608
+- Add `CODEOWNERS` file for auto assigning reviewer by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/605
+
+### Docs
+
+- Fix typo in `docs/zh_cn/tutorials/model.md` by @C1rN09 in https://github.com/open-mmlab/mmengine/pull/598
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:21]: Put typos at the end of the section.

File: docs/en/notes/changelog.md [hunk 3]
```diff
@@ -1,5 +1,63 @@
...
+- Support running with NPU backend. by @wangjiangben-hw in https://github.com/open-mmlab/mmengine/pull/572
+- Support torch `ZeroRedundancyOptimizer` by @nijkah in https://github.com/open-mmlab/mmengine/pull/551
+- Add non-blocking feature to `BaseDataPreprocessor` by @shenmishajing in https://github.com/open-mmlab/mmengine/pull/618
+- Add documents for `clip_grad` , and support clip grad by value. by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/513
+- Add ROCm info when collecting env by @zhouzaida in https://github.com/open-mmlab/mmengine/pull/633
+- Add a function to mark the deprecated function. by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/609
+- Call `register_all_modules`  in `Registry.get()` by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/541
+- Deprecate `_save_to_state_dict` implemented in mmengine by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/610
+- Add `ignore_keys` in ConcatDataset by @BIGWangYuDong in https://github.com/open-mmlab/mmengine/pull/556
+- Add issue or pr assignees by @zhouzaida in https://github.com/open-mmlab/mmengine/pull/608
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:16]: unnecessary, delete this

File: docs/en/notes/changelog.md [hunk 4]
```diff
@@ -1,5 +1,63 @@
...
+- Add `CODEOWNERS` file for auto assigning reviewer by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/605
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:17]: unnecessary

**Review by ZwwWayne — APPROVED** [review]

File: docs/en/notes/changelog.md [hunk 5]
```diff
@@ -1,5 +1,61 @@
 # Changelog of v0.x
 
+## v0.3.0 (11/02/2022)
+
+### New Features & Enhancements
+
+- Support running with NPU backend. by @wangjiangben-hw in https://github.com/open-mmlab/mmengine/pull/572
```
Comment (zhouzaida) [inline docs/en/notes/changelog.md:7]:
```suggestion
- Support running on Ascend chip by @wangjiangben-hw in https://github.com/open-mmlab/mmengine/pull/572
```

File: docs/en/notes/changelog.md [hunk 6]
```diff
@@ -1,5 +1,61 @@
...
+- Support torch `ZeroRedundancyOptimizer` by @nijkah in https://github.com/open-mmlab/mmengine/pull/551
+- Add non-blocking feature to `BaseDataPreprocessor` by @shenmishajing in https://github.com/open-mmlab/mmengine/pull/618
+- Add documents for `clip_grad` , and support clip grad by value. by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/513
```
Comment (zhouzaida) [inline docs/en/notes/changelog.md:10]:
```suggestion
- Add documents for `clip_grad`, and support clip grad by value. by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/513
```

File: README_zh-CN.md [hunk 7]
```diff
@@ -60,14 +60,13 @@ MMEngine 是一个基于 PyTorch 用于深度学习模型训练的基础库，
...
 ## 最近进展
 
-最新版本 v0.2.0 在 2022.10.11 发布。
+最新版本 v0.3.0 在 2022.11.02 发布。
 
-1. 重构 FileIO 以提供更加易用的接口并保持向下兼容。
-2. 新增 Test time augmentation 模型基类。
-3. 分布式训练时，支持将 BN 自动转化为 SyncBN。
-4. 新增了 SMDDP 后端并支持在 AWS 进行分布式训练。
+1. 支持 NPU 后端。
```
Comment (zhouzaida) [inline README_zh-CN.md:65]:
```suggestion
1. 支持华为昇腾芯片。
```

File: README.md [hunk 8]
```diff
@@ -58,14 +58,13 @@ Major features:
...
 ## What's New
 
-v0.2.0 was released in 2022-10-11.
+v0.3.0 was released in 2022-11-02.
 
-1. Refactor FileIO to provide a more unified interface without BC breaking.
-2. Add the base model for test time augmentation.
-3. Support converting `BN` to `SyncBN` during distributed data-parallel training.
-4. Add SMDDP backend and support running on AWS.
+1. Support NPU backend.
```
[the remark this answers is not in the document]
Comment (zhouzaida) [reply README.md:63]: Support Ascend chip.

**Review by zhouzaida — APPROVED** [review]

**Pull request merged**